src/test/features/response/tasks/whenWillYouPayTask.ts

```diff
@@ -0,0 +1,27 @@
+import { expect } from 'chai'
+
+import { WhenWillYouPayTask } from 'response/tasks/whenWillYouPayTask'
+
+import { ResponseDraft } from 'response/draft/responseDraft'
+import { DefendantPaymentType, DefendantPaymentOption } from 'response/form/models/defendantPaymentOption'
+
+describe('WhenWillYouPayTask', () => {
+  describe('when no response', () => {
```

Comment:
This test scenario is totally useless. You prepare data by setting response to undefined while `WhenWillYouPayTask.isCompleted` checks `responseDraft.defendantPaymentOption && responseDraft.defendantPaymentOption.option` This test is green by your lack.

Reply:
Ok will take a look and sort it out

Reply:
Updated test cases